server/data-service/README.md

```diff
@@ -47,7 +47,7 @@ To start the service
    | | | input json example when it is a result dataset example (parameter name : dataset, item type: text) |
    | | | { dataType: "http://localhost:8080/semantics/edu.illinois.ncsa.ergo.eq.schemas.buildingDamageVer4.v1.0", title: "shelby building damage", sourceDataset: "59e5098168f47426547409f3", format: "csv"} |
    | | | input json example when it is a parent network dataset (parameter name: dataset, item type: text) |
-   | | | { dataType: "http://localhost:8080/semantics/edu.illinois.ncsa.ergo.eq.schemas.buildingInventoryVer4.v1.0", title: "Shelby_County_Essential_Facilities", sourceDataset: "", format: "shp-network", networkDataset:{link:{linkType:  "pipeline"}, node:{networkType: "water facility"}, graph:{graphType: "table"}} |
+   | | | { dataType: "http://localhost:8080/semantics/edu.illinois.ncsa.ergo.eq.schemas.buildingInventoryVer4.v1.0", title: "Shelby_County_Essential_Facilities", sourceDataset: "", format: "shp-network", networkDataset:{link:{dataType:  "pipeline"}, node:{dataType: "water facility"}, graph:{dataType: "table"}} |
```

Comment:
Do we have a real example we can put here instead of a made up one? I think this is misleading because the overall type is "buildingInventoryVer4v1.0" and that's not a network at all. Something like what's on the wiki page you referenced? Also, I believe the graph dataType shouldn't be table since we have a format called table, it's confusing. Let me know if the below makes sense or not. I used what was on the wiki and made up a type for the graph.

```json
{
   "dataType":"incore:network",
   "title":"Centerville epn network",
   "sourceDataset":"",
   "format":"shp-network",
   "networkDataset":{
      "link":{
         "dataType":"incore:epnLinkeVer1"
      },
      "node":{
         "dataType":"incore:epnNodeVer1"
      },
      "graph":{
         "dataType":"incore:networkGraph"
      }
   }
}
```

Reply:
That is good finding I will update